datafusion-expr/src/expr.rs

```diff
@@ -399,7 +399,7 @@ impl fmt::Debug for Expr {
         match self {
             Expr::Alias(expr, alias) => write!(f, "{:?} AS {}", expr, alias),
             Expr::Column(c) => write!(f, "{}", c),
-            Expr::ScalarVariable(var_names) => write!(f, "{}", var_names.join(".")),
+            Expr::ScalarVariable(_, var_names) => write!(f, "{}", var_names.join(".")),
```

Comment:
I avoided this because it looks like the Debug implementation for Expr tries to print SQL-compatible output, but, for example, "@v0, data type: DataType::Utf8" wouldn't be SQL-like

Reply:
I avoided this because it looks like the Debug implementation for Expr tries to print SQL-compatible output, but, for example, "@v0, data type: DataType::Utf8" wouldn't be SQL-like

Reply:
> it looks like the Debug implementation for Expr tries to print SQL-compatible output

It's a good point, makes sense to me.